From 55f99f3ea14aebfa0f6faf52b0b0a81e2a5c1d8b Mon Sep 17 00:00:00 2001
From: Richard Russon <rich@flatcap.org>
Date: Mon, 6 Nov 2017 13:19:57 +0000
Subject: [PATCH] autosetup: fix check for missing sendmail

If sendmail wasn't on the path, then configure.autosetup defined
    SENDMAIL=false
---
 auto.def | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/auto.def b/auto.def
index 31df9764..6a59b9bb 100644
--- a/auto.def
+++ b/auto.def
@@ -188,8 +188,7 @@ if {1} {
   # Check for tools and programs
   cc-check-tools ar ranlib strip
   cc-check-progs install
-  cc-path-progs sendmail
-  if {![is-defined SENDMAIL]} {
+  if {![cc-path-progs sendmail]} {
     define SENDMAIL /usr/sbin/sendmail
   }
 
-- 
2.11.0

